viewer-prototype/src/browser/trace-viewer/components/utils/filtrer-tree/tree-node.tsx

```diff
@@ -55,7 +55,7 @@ export class TreeNodeComponent extends React.Component<TreeNodeComponentProps> {
                     { this.isLeaf()
                         ? <span style={{paddingLeft:this.props.padding}}></span>
                         : <span
-                            onClick={() => this.props.onCollapsed(this.props.node.id)}
+                            onClick={() => this.props.onToggleCollapse(this.props.node.id)}
```

Comment:
`According` to the coding guidelines under the react section shouldn't this line something like the following:

    render(): JSX.Element {
        return <div onClick={this.onClickDiv} >
            <span style={{padding: 5}}>
                {this.renderCheckbox(this.props.checkedStatus)}
            </span>
            {this.props.name}
        </div>;
    }

    protected onClickDiv = () => {
        this.props.onToggleCheck(this.props.id)
    }


[1] https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines